src/pages/TodosPage.jsx

```diff
@@ -0,0 +1,190 @@
+import styles from './TodosPage.module.css';
```

Comment:
interesting choice to bring your fetch logic to `TodosPage`. I do agree that it doesn't belong in app and it is more suitable in TodosPage. Good judgement! In future implementations you'll want to create its own requests module or folder to hold all api operations.